### PR TITLE
DOCS-7748 Continuous Testing & Getting Started with Continuous Testing Doc Updates

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Updates mentions of the left-hand product navigation menu in the Continuous Testing and Getting Started with Continuous Testing documentation (as a start). Replaces screenshots with the new navigation and updates UI elements based on recent product changes.

Remember to set the `cache_enabled: false` to `true` in the `pull_config_preview.yaml` file before merging!

### Merge instructions

- [] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_testing/_index.md [hunk 1]
```diff
@@ -39,19 +39,21 @@ Datadog Continuous Testing offers a set of tools that enable you to automate sof
 
 ## Test with ease and speed
 
-Use scalable features such as a codeless [web recorder][2], [mobile app recorder][15], [parallel test runs][3], and built-in multi-location testing to save time and effort for your QA team.
+Use scalable features such as a codeless [web recorder][2], [mobile app recorder][15], [parallel test runs][3], and built-in multi-location testing to save time and effort for your QA team. You can run your tests sequentially and customize the number of tests you want to run at the same time on the [**Settings** page][3].
```
Comment (urseberry) [inline content/en/continuous_testing/_index.md:42]: Is the [3] link to parallel test runs correct? I'm surprised it goes to the same place as the Settings page.

**Review by urseberry — COMMENTED** [review]
Realized I don't have enough time to finish this review today. Converting this to an editorial review.

File: content/en/continuous_testing/_index.md (hunk 1 above)
Comment (alai97) [reply content/en/continuous_testing/_index.md:42]: Yes, it is. Users set their test parallelization limits on the [Settings page](https://docs.datadoghq.com/continuous_testing/settings/#parallelization).

File: content/en/continuous_testing/_index.md [hunk 2]
```diff
@@ -39,19 +39,21 @@ Datadog Continuous Testing offers a set of tools that enable you to automate sof
...
 
-{{< img src="continuous_testing/continuous_testing_selection.png" alt="Choose between running your tests sequentially and customizing the number of tests you want to run at the same time in the Continuous Testing Settings page" style="width:100%;">}}
+{{< img src="continuous_testing/settings/parallelization.png" alt="Choose between running your tests sequentially and customizing the number of tests you want to run at the same time in the Continuous Testing Settings page" style="width:100%;">}}
```
Comment (aliciascott) [inline content/en/continuous_testing/_index.md:44]:
```suggestion
{{< img src="continuous_testing/settings/parallelization.png" alt="Choose between running your tests sequentially and customizing the number of tests you want to run at the same time in the Continuous Testing Settings page" style="width:100%;">}}
```

Just a nit but could you possibly re-take the screen shot to remove that one inch section of blank space and gray bar on the right hand side ?

File: content/en/continuous_testing/_index.md [hunk 3]
```diff
@@ -61,19 +63,23 @@ You can use the [Datadog Terraform provider][10] to control test creation and st
...
 
 ## Ready to start?
 
-After you have configured some [Synthetic tests][4], see the documentation for your preferred [CI/CD provider][1] or use the [datadog-ci NPM package][14] in your CI/CD pipelines. See [Testing Local and Staging Environments][17] to use Continuous Testing in environments that are not publicly available or production, for example, running tests against your local development environment or a staging environment within a private network. Then, start exploring details about your batch runs in the [Synthetic Monitoring & Continuous Testing Explorer][11].
+After you have configured some [Synthetic tests][4], see the documentation for your preferred [CI/CD provider][1] or use the [`datadog-ci` NPM package][14] in your CI/CD pipelines. See [Testing Local and Staging Environments][17] to use Continuous Testing in environments that are not publicly available or production, for example, running tests against your local development environment or a staging environment within a private network. Then, start exploring details about your batch runs in the [Synthetic Monitoring & Testing Results Explorer][11].
```
Comment (aliciascott) [inline content/en/continuous_testing/_index.md:82]:
```suggestion
After you have configured some [Synthetic tests][4], see the documentation for your preferred [CI/CD provider][1], or use the [`datadog-ci` NPM package][14] in your CI/CD pipelines. See [Testing Local and Staging Environments][17] to use Continuous Testing in environments that are not publicly available or production, for example, running tests against your local development environment or a staging environment within a private network. Then, start exploring details about your batch runs in the [Synthetic Monitoring & Testing Results Explorer][11].
```

File: content/en/continuous_testing/explorer/_index.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ You can accomplish the following actions:
 
 ## Create a search query
 
-Navigate to [**UX Monitoring** > **Explorer**][1] and click on an out-of-the-box search query to start viewing your test batches or runs and creating visualizations.
+Navigate to [**Digital Experience > Synthetic Monitoring & Testing** > **Continuous Testing**][1] and click on an out-of-the-box search query to start viewing your test batches or runs and creating visualizations.
```
Comment (aliciascott) [inline content/en/continuous_testing/explorer/_index.md:45]:
```suggestion
Navigate to [**Digital Experience > Synthetic Monitoring & Testing** > **Continuous Testing**][1] and click on an out-of-the-box search query to start viewing your test batches or runs and create visualizations.
```

File: content/en/continuous_testing/settings/_index.md [hunk 5]
```diff
@@ -25,9 +25,9 @@ further_reading:
 
 ## Overview
 
-You can access Continuous Testing settings on the [Synthetic Monitoring & Continuous Testing Settings page][1].
+You can access Continuous Testing settings on the [Synthetic Monitoring & Testing Settings page][1].
 
-{{< img src="continuous_testing/continuous_testing_settings_default.png" alt="Default settings for Continuous Testing" style="width:100%;">}}
+{{< img src="continuous_testing/settings/parallelization.png" alt="Set parallelization for your Continuous Testing tests on the Settings page" style="width:100%;">}}
```
Comment (aliciascott) [inline content/en/continuous_testing/settings/_index.md:30]:
```suggestion
{{< img src="continuous_testing/settings/parallelization.png" alt="Set parallelization for your Continuous Testing tests on the Settings page" style="width:100%;">}}
```
same nit here with this image, can you re-crop it to remove the white space and gray bar on the right ?

**Review by aliciascott — APPROVED** [review]
Hey @alai97 this looks great! Thank you so much for being so thorough with these updates! Just a couple small nit things, but approved!

File: static/images/continuous_testing/explorer/results_explorer.png
Comment (etnbrd) [inline static/images/continuous_testing/explorer/results_explorer.png:1]: Nit: do we want to take this screenshot without the shadow at the bottom?
It seems a bit odd, but I'm not sure what it is and if it's desired or not.

**Review by etnbrd — APPROVED** [review]
LGTM for CT 👍

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```

**Pull request merged**